plugins/org.polarsys.capella.cybersecurity.doc/html/usermanual.mediawiki

```diff
@@ -97,7 +107,10 @@ The attributes of a Primary Asset are its corresponding Security Needs, i.e. the
 * Availability: property of being accessible and useable upon demand by an authorized entity.
 * Traceability: property of verifying access and use thanks to documented recorded identification.
 
-An asset can define a value between 0 and 3 for each of these attributes. Project/Product teams shall define the equivalences between these sensitivity levels and those specific to their standards framework.
+By default, an asset can define a value between 0 and 3 for each of these attributes. Project/Product teams shall define the equivalences between these sensitivity levels and those specific to their standards framework.
+If this values are not suitable for the project, the user could customize (remove, add, rename) them from Cybersecurity Configuration.

```

Comment:
This element is located under the Project element which is hidden by default,

Can you add something like: 

If this values are not suitable for the project, the user could customize (remove, add, rename) them from Cybersecurity Configuration located on the root Project element (see Capella Guide > Preferences > Project Explorer to reveal it)

Reply:
Malina, add also this part:  located on the root Project element (see Capella Guide > Preferences > Project Explorer to reveal it)